requirements.txt

```diff
@@ -1,2 +1,3 @@
-projectq
+git+https://github.com/ProjectQ-Framework/ProjectQ.git#egg=projectq
```

Comment:
This will always use the latest commit to `master`, no? What if there is a future long-lived commit in `master` that is broken? Would it be better to pick a particular release/commit?

Reply:
Unfortunately the ProjectQ IBM API fix was only committed 5 days ago. This change is so that we can work on this PR and verify we can get the plugin working with the IBM backend; we should re-evaluate the status of ProjectQ before merging this in.

But good call on pinning to a particular commit :+1:

Reply:
It looks like this is going to be released soon as v0.5.1

Reply:
https://github.com/ProjectQ-Framework/ProjectQ/pull/370

Reply:
Great!